src/components/Toast/index.stories.tsx

```diff
@@ -95,13 +118,72 @@ function Persistent() {
   )
 }
 
+function PersistentWithAdornment() {
+  const { notify } = useToast()
+
+  return (
+    <>
```

Comment:
Maybe nice to wrap this in a div so the buttons have a bit of spacing between them 🙈